.github/workflows/deployment-test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
   deployment-test:
     needs: pre_check
     if: needs.pre_check.outputs.should_run == 'true'
-    runs-on: ubuntu-18.04 # trusty
+    runs-on: NewAutoScaled
```

Comment:
Yeah that's gonna be one of the complications. For various reasons must still run the deployment test on an outdated version of Ubuntu. Last time @michaelfig and I tried to update we ran into trouble and decided to punt.